src/registry.js

```diff
@@ -1,7 +1,12 @@
 const fs = require('fs');
 const validateJson = require('./validate-json-schema');
 
-const REGISTRY_JSON_FILE = './registry.json';
+const REGISTRY_JSON_FILE = 'registry.json';
+
+const TEMPLATE_STATUS_IN_VERIFICATION = 'InVerification';
+const TEMPLATE_STATUS_APPROVED = 'Approved';
+const TEMPLATE_STATUS_REJECTED = 'Rejected';
```

Comment:
Sorry, what is difference between Rejected and Error? Maybe make sense to add some comment about it.

Reply:
I mentioned that on https://wiki.corp.adobe.com/display/dxdevx/Template+Registry+API and in the [OpenAPI Schema](https://github.com/magento-eqp/aio-template-registry-api/blob/187d38107e0ca4411cf669ad35f86d62fbbed69a/openapi.yaml#L333). "Error" indicates that an application error occurred. It might be an issue with Github itself, for example.